pkg/controllers/managementuser/rbac/project_handler.go

```diff
@@ -156,6 +171,7 @@ func (p *pLifecycle) assignNamespacesToProject(project *v3.Project, projectName
 		}
 		ns.Annotations[projectIDAnnotation] = fmt.Sprintf("%v:%v", p.m.clusterName, project.Name)
 		if _, err := p.m.workload.Core.Namespaces(p.m.clusterName).Update(ns); err != nil {
+			log.Errorf("projectHandler: assignNamespacesToProject: error adding project id annotation to namespace err=%v", err)
```

Comment:
The `err` will get logged by the handler if you return it, so it doesn't need to be logged here.